src/highlight/HighlightCanvas.tsx

```diff
@@ -142,3 +143,7 @@ export default class HighlightCanvas extends React.Component<Props> {
         return <canvas ref={this.canvasRef} className="ba-HighlightCanvas" />;
     }
 }
+
+export { HighlightCanvas as HighlightCanvasComponent };
+
+export default React.memo(HighlightCanvas);
```

Comment:
This is typically used for function components. Should `HighlightCanvas` be made a PureComponent, instead?